docs/jwst/associations/generator.rst

```diff
@@ -90,4 +90,11 @@ associations depending on the candidates.
 Association Candidates
 ----------------------
 
-TBD
+An association candidate has identifier of ``c1NNN`` in the pool.
```

Comment:
This is copied over from Table 1 in https://jwst-docs.stsci.edu/accessing-jwst-data/jwst-science-data-overview that ironically tries to link back to the RTD doc of this package.

Reply:
In the context of this RtD page, I believe this `Association Candidates` section should include all asn candidate types, e.g. both lines in Table 1 are association candidate types. Unfortunately one type of association candidate is "candidate". 😬 

In the context of association generation, it views observation associations as "association candidates" during rule checking to see if they should be generated. We should probably include both here.

Thoughts @stscieisenhamer ?

Reply:
Both should be included.

Yes, it was an unfortunate collision of names that the set of association candidates include association candidates.

For documentation, I don't recall whether there is documentation or not; if there were it would be either here or in the jwst level2/3 technical pages.

If not, the candidate type "candidate" are association candidates created by APT. These are associations defined by APT depending on observing modes and templates used. These candidates are suggested candidates. Once the proposal is executed, the generator may or may not create all suggested candidates. Some reasons a candidate is not used are the results would have no science value, insufficient exposures taken due to observatory issues, duplicate products would be produced, etc.

Reply:
Thanks! I added more text as you suggested. Does it look better?